apps/agreement/contracts/Agreement.sol

```diff
@@ -452,7 +452,6 @@ contract Agreement is IArbitrable, ILockManager, IAgreement, IACLOracle, AragonA
     *      Can only be called once (if at all) per opened challenge.
     *      Initialization check is implicitly provided by `_getChallengedAction()` as disputable actions can only be created via `newAction()`.
     * @param _actionId Identification number of the action to be disputed
-    // Note: this may be confusing; perhaps we should flip the concept to "submitter would like to submit more?"
```

Comment:
Last note: as discussed offline, we have a simplification in mind for submitting evidence :).